config/packages/app.yaml

```diff
@@ -0,0 +1,25 @@
+services:
```

Comment:
Yes, I removed it ... but whenever you run `composer install`, the file is recreated, so I prefer to keep it until Symfony removes it officially.